ClusterTest/project/assembly.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.15.0")
```

Comment:
For my learning purpose, what's this plug-in for?

Reply:
This is for running `sbt assembly`. For convenience purposes when running `spark-submit` in our stress infrastructure, we bundle all of the dependencies into fat JARs; assembly builds the JARs with all dependencies included.

Reply:
Thanks!